packages/now-cli/src/util/dev/builder.ts

```diff
@@ -84,7 +73,6 @@ async function createBuildProcess(
     },
     execPath,
     execArgv: [],
-    stdio: ['ignore', 'pipe', 'pipe', 'ipc'],
```

Comment:
Is inherit the default stdio? I was looking for where 'inherit' was set 🤔

Reply:
Ya it's the default for `fork()`.